kafka-ui-api/src/main/java/com/provectus/kafka/ui/cluster/config/ClustersProperties.java

```diff
@@ -0,0 +1,26 @@
+package com.provectus.kafka.ui.cluster.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Configuration
+@ConfigurationProperties("kafka")
+@Data
+public class ClustersProperties {
+
+    List<Cluster> clusters = new ArrayList<>();
+
+    @Data
+    public static class Cluster {
+        String id;
+        String name;
+        String bootstrapServers;
```

Comment:
Maybe list here is more suitable

Reply:
didn't get the idea. We need to load list of clusters, so we have list of clusters in cluster properties com.provectus.kafka.ui.cluster.config.ClustersProperties#clusters , and we need to load data to them from configs, so this Cluster DTO for properties is used. Could you clarify where list can be used?

Reply:
I think we can have bootstrapServers as string, as it will be simpler to pass it as config if we need it later, and kafka API also wants String